UI/xdg-data/com.obsproject.Studio.appdata.xml.in

```diff
@@ -9,14 +9,30 @@
   <summary>Live streaming and video recording software</summary>
   <description>
     <p>Free and open source software for video capturing, recording, and live streaming.</p>
+    <p>Features</p>
+    <ul>
+      <li>High performance real time video/audio capturing and mixing. Create scenes made up of multiple sources including window captures, images, text, browser windows, webcams, capture cards and more.</li>
+      <li>Set up an unlimited number of scenes you can switch between seamlessly via custom transitions.</li>
+      <li>Intuitive audio mixer with per-source filters such as noise gate, noise suppression, and gain. Take full control with VST plugin support.</li>
+      <li>Powerful and easy to use configuration options. Add new Sources, duplicate existing ones, and adjust their properties effortlessly.</li>
+      <li>Streamlined Settings panel gives you access to a wide array of configuration options to tweak every aspect of your broadcast or recording.</li>
+      <li>Modular 'Dock' UI allows you to rearrange the layout exactly as you like. You can even pop out each individual Dock to its own window.</li>
+    </ul>
+    <p>Create Professional Productions</p>
```

Comment:
I'm not sure this is worth a paragraph. Should both lists be merged?

Reply:
I based myself on the layout of the main landing page; https://obsproject.com/ In there you also have two lists.